pkg/test/resourcefixture/testdata/basic/compute/v1alpha1/computemanagedsslcertificate/managedsslcertificate/_http.log

```diff
@@ -127,7 +127,7 @@ X-Xss-Protection: 0
   "selfLink": "https://www.googleapis.com/compute/beta/projects/${projectId}/global/operations/${operationID}",
   "startTime": "2024-04-01T12:34:56.123456Z",
   "status": "RUNNING",
-  "targetId": "7780111738275460050",
+  "targetId": "1719033726205916358",
```

Comment:
I don't get why this one wasn't replaced.  Looking at the code it should have been, I think.  Maybe/hopefully you just need to recapture the golden output here?

Reply:
😆 ah, yes. I reminded myself to recapture the golden log while waiting for another full fixtures run. It took too long and I totally forgot this.